fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/NopObject.java

```diff
@@ -33,6 +33,11 @@ public void apply(IDatabaseAdapter target) {
         // We're NOP so we do nothing on purpose
     }
 
+    @Override
+    public void apply(Integer priorVersion, IDatabaseAdapter target) {
```

Comment:
might as well push this into the interface as a default implementation.

Reply:
didn't end up doing this one but can if you think its important.
instead, i opted to add "ThrowUnsupportedOperationException" in cases where we're trying to update objects that we shouldn't (i.e. tablespace and sequence).